signer/src/testing/transaction_coordinator.rs

```diff
@@ -213,6 +211,42 @@ where
             prevouts: Vec::new(),
         };
         test_data.push_bitcoin_txs(&bitcoin_chain_tip, vec![tx_info], &signer_script_pubkeys);
+
+        // Also ensure one valid withdrawal exists for test consistency
+        let stacks_blocks = test_data
+            .stacks_blocks
+            .iter()
+            .filter_map(|b| {
+                if b.bitcoin_anchor == bitcoin_chain_tip.block_hash {
+                    Some(b.block_hash)
+                } else {
+                    None
+                }
+            })
+            .collect::<HashSet<_>>();
+        let mut withdrawal = test_data
+            .withdraw_requests
+            .iter()
+            .find(|w| stacks_blocks.contains(&w.block_hash))
+            .unwrap()
+            .clone();
+
+        let mut withdrawal_votes = test_data
+            .withdraw_signers
+            .iter()
+            .filter(|ws| ws.qualified_id() == withdrawal.qualified_id())
+            .cloned()
+            .collect::<Vec<_>>();
+
+        withdrawal.request_id *= 100;
```

Comment:
Is zero a valid withdrawal `request_id` here? Does it matter? I think the answer is "no" for this test since we get the withdrawal request from a block anchored to the bitcoin chain tip so it is unlikely to be first withdrawal.

Also, can't wait for us to not have to do this after doing https://github.com/stacks-sbtc/sbtc/issues/1671.

Reply:
It shouldn't be possible to get zero here since `TestData` uses incremental IDs starting from 1. But yeah, really looking forward to a more sane approach for `TestData` 😅